lesson_13/maps_java/maps_app/src/main/java/com/codedifferently/lesson13/Lesson13.java

```diff
@@ -1,12 +1,25 @@
 package com.codedifferently.lesson13;
 
-public class Lesson13 {
+import java.util.HashMap;
 
-  /**
-   * Provide the solution to LeetCode 3146 here:
-   * https://leetcode.com/problems/permutation-difference-between-two-strings
-   */
+public class Lesson13 {
   public int findPermutationDifference(String s, String t) {
-    return 0;
+    HashMap<Character, Integer> letters = new HashMap<>();
```

Comment:
Could have a better variable name. Try to come up with something more descriptive.